src/components/todo/TodoInput.js

```diff
@@ -1,10 +1,54 @@
-import React, {Component} from 'react';
-import styled from 'styled-components';
+import React, { Component } from "react";
+import styled from "styled-components";
+
+const Row = styled.div`
```

Comment:
이 부분을
```suggestion
const Row = styled.form`
```
으로 바꿨을때 가질 수 있는 장점과 단점에 대해서 생각해보시면 좋을 것 같아요.